src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Microsoft.Extensions.Configuration.Binder.SourceGeneration.Tests.csproj

```diff
@@ -48,8 +48,15 @@
     <ProjectReference Include="..\..\gen\Microsoft.Extensions.Configuration.Binder.SourceGeneration.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="true" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
-    <Content Include="Baselines\*.generated.txt;Baselines\ConfigurationBinder\*.generated.txt;Baselines\OptionsBuilder\*.generated.txt;Baselines\ServiceCollection\*.generated.txt">
+  <ItemGroup>
+    <Content Include="Baselines\net462\*.generated.txt;
```

Comment:
>"Baselines [](http://example.com/codeflow?start=21&length=10)

do we need to copy specific files? can't we include the whole Baseline folder?

Reply:
Include just Baselines does not seem to work, the max I could do is this lightweight version :
```
<Content Include="Baselines\*\*\*.generated.txt;
         Baselines\*\*.generated.txt;">
```